src/test/java/org/broadinstitute/dsde/consent/ontology/datause/DataUseMatcherTest.java

```diff
@@ -1,7 +1,7 @@
 package org.broadinstitute.dsde.consent.ontology.datause;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.mindswap.pellet.utils.ATermUtils.assertTrue;
```

Comment:
Is this intentionally not being imported from junit?

Reply:
Ah ... good catch, no, this is unintentional.

Reply:
There were several of these, all are updated now.